x/avs/keeper/avs.go

```diff
@@ -59,3 +77,66 @@ func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context) ([]string, error) {
 
 	return avsAddrList, nil
 }
+
+func (k *Keeper) GetAVSAddrByChainID(ctx sdk.Context, chainID string) (string, error) {
+	chainID = ProcessingStr(chainID)
+	if len(chainID) == 0 {
+		return "", errorsmod.Wrap(types.ErrNotNull, "SetAVSAddrByChainID: chainID is null")
+	}
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixAVSInfoByChainID)
+	if !store.Has([]byte(chainID)) {
+		return "", errorsmod.Wrap(types.ErrNoKeyInTheStore, fmt.Sprintf("GetAVSAddrByChainID: key is %s", chainID))
+	}
+	avsAddr := store.Get([]byte(chainID))
+
+	return string(avsAddr), nil
+}
+
+// SetAVSAddrByChainID creates an avs address given the chainID
+func (k Keeper) SetAVSAddrByChainID(ctx sdk.Context, chainID string) (err error) {
+	chainID = ProcessingStr(chainID)
+	if len(chainID) == 0 {
+		return errorsmod.Wrap(err, "SetAVSAddrByChainID: chainID is null")
+	}
+	avsAddr := common.BytesToAddress(Keccak256([]byte(chainID))).String()
```

Comment:
No need to copy Keccak256 code here; just use `"github.com/ethereum/go-ethereum/crypto"` ?
```suggestion
	avsAddr := common.BytesToAddress(
	    crypto.Keccak256(
	        []byte{0xFE}, []byte(chainID),
	    )[12:],
	).String()
```

Reply:
fixed